### PR TITLE
feat(serve): manual mode integration

### PR DESCRIPTION
**Pull request by pcattori**
Testing Strategy:

Tested locally via `LOCAL_BUILD_DIRECTORY`

## TODO

- [x] Windows: URL file path for `import`
- [x] kill app server and esbuild after tests run

### CODE REVIEW

File: packages/remix-serve/cli.ts [hunk 1]
```diff
@@ -32,9 +33,41 @@ async function run() {
     process.exit(1);
   }
 
-  let buildPath = url.pathToFileURL(
-    path.resolve(process.cwd(), buildPathArg)
-  ).href;
+  let buildPath = path.resolve(buildPathArg);
```
Comment (jacob-ebey) [inline packages/remix-serve/cli.ts:36]: This will fail on windows, you need the url.PathToFileURL

File: templates/remix-javascript/package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "type": "module",
   "scripts": {
     "build": "remix build",
-    "dev": "remix dev",
+    "dev": "remix dev --manual",
```
Comment (jacob-ebey) [inline templates/remix-javascript/package.json:7]: Can we default to manual if no command is provided now that the serve CLI that will be used does cache busting?

**Review by jacob-ebey — CHANGES_REQUESTED** [review]

File: templates/remix-javascript/package.json (hunk 2 above)
Comment (pcattori) [reply templates/remix-javascript/package.json:7]: We could, but then we'd need a way to opt out of `--manual` which we don't currently have.

For v2, Michael and I agreed that we probably want a lever like `--mode=watch`, `--mode=live-reload`, `--mode=hot` or something like that. naming TBD

Comment (jacob-ebey) [reply templates/remix-javascript/package.json:7]: I just mean:
```js
if (!userProvidedCommand) {
  manual = true
}
```

**Pull request merged**